paper-tabs.html

```diff
@@ -310,6 +310,10 @@
       'iron-deselect': '_onIronDeselect'
     },
 
+    ready: function() {
+      this.setScrollDirection('y', this.$.tabsContainer);
```

Comment:
This turns off `track` if you try to drag in the vertical axis. I went over this slightly in my talk :no_mouth: